dev-support/hbase-vote.sh

```diff
@@ -164,11 +167,11 @@ function print_when_exit() {
         * Signature: $( ((SIGNATURE_PASSED)) && echo "ok" || echo "failed" )
         * Checksum : $( ((CHECKSUM_PASSED)) && echo "ok" || echo "failed" )
         * Rat check (${JAVA_VERSION}): $( ((RAT_CHECK_PASSED)) && echo "ok" || echo "failed" )
-         - mvn clean apache-rat:check
+         - mvn clean apache-rat:check "${MVN_PROPERTIES}"
         * Built from source (${JAVA_VERSION}): $( ((BUILD_FROM_SOURCE_PASSED)) && echo "ok" || echo "failed" )
-         - mvn clean install -DskipTests
+         - mvn clean install -DskipTests "${MVN_PROPERTIES}"
         * Unit tests pass (${JAVA_VERSION}): $( ((UNIT_TEST_PASSED)) && echo "ok" || echo "failed" )
-         - mvn package ${MVN_ARGS}
+         - mvn package ${MVN_PROFILES} "${MVN_PROPERTIES}"
```

Comment:
[nit] should we add `-Dsurefire.rerunFailingTestsCount=3` as part of the output?

Reply:
We can if you'd like. Since it's a "standard" part of the script, I don't think there's much value in adding it. I also wonder if it should be removed from the mvn invocation; in my experience, the tests that are flakey are cluster tests, and cluster tests tend to not clean up after themselves well enough to have a surefire rerun launch in a valid environment -- usually the retries fail due to "unable to start cluster" or similar setup nonsense.